core/src/forward_packet_batches_by_accounts.rs

```diff
@@ -0,0 +1,335 @@
+use {
+    crate::unprocessed_packet_batches::{self, ImmutableDeserializedPacket},
+    solana_perf::packet::Packet,
+    solana_runtime::{
+        bank::Bank,
+        block_cost_limits,
+        cost_tracker::{CostTracker, CostTrackerError},
+    },
+    solana_sdk::pubkey::Pubkey,
+    std::{rc::Rc, sync::Arc},
+};
+
+/// `ForwardBatch` to have half of default cost_tracker limits, as smaller batch
+/// allows better granularity in interating through buffer, and faster forwarding.
```

Comment:
> Why does this allow for faster forwarding? Isn't the number of transactions being included roughly the same?

Good point. Thought smaller batches would go through forwarding pipeline smoother, has less chance to hit line rate limit. But think again, it doesn't really matter. It actually adds overhead of checking data_budget, and metrics. Tho not in significant way. Removed this comment to avoid confusion.